app/models/north_utility.rb

```diff
@@ -1,2 +1,9 @@
 class NorthUtility < Utility
+  def short_content
+    50
+  end
+
+  def medium_content
```

Comment:
```suggestion
  def medium_content_length
```
Acá también puede ir